### PR TITLE
(Blog) 2021-02-19 Status Update

### PR DESCRIPTION
**Pull request by carabasdaniel**

### CODE REVIEW

File: _posts/2021-02-19-status-update.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  - PDKSync
+  - pdk-templates
+  - pdk
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-apache#2122`][puppetlabs-apache-pr-2122]: "(IAC-1186) Add $use_port_for_filenames parameter", thanks to [smortex][smortex]
```
Comment (sanfrancrisko) [inline _posts/2021-02-19-status-update.md:22]: This PR was dropped off the list as it was authored by me, but it actually had contributions from two community members too and it's a customer defect:
```suggestion
- [`puppetlabs-apache#2122`][puppetlabs-apache-pr-2122]: "(IAC-1186) Add $use_port_for_filenames parameter", thanks to [smortex][smortex]
- [`puppetlabs-apache#2121`][puppetlabs-apache-pr-2121]: "(MODULES-10899) Handle PHP8 MOD package naming convention changes", thanks to [cdamage][cdamage] for the original fix, [beatchristen][beatchristen] for some enhancements for Debian 10 and [gguillotte][gguillotte] for reviewing.
```

Comment (sanfrancrisko) [reply _posts/2021-02-19-status-update.md:22]: Here's the links required too:
```md
[beatchristen]: https://github.com/beatchristen
[cdamage]:       https://github.com/cdamage
[gguillotte]:       https://github.com/gguillotte
[puppetlabs-apache-pr-2121]: https://github.com/puppetlabs/puppetlabs-apache/pull/2124
```

File: _posts/2021-02-19-status-update.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- [`puppetlabs-apt`][puppetlabs-apt] (`7.7.1`)
+- [`puppetlabs-apache`][puppetlabs-apache] (`5.10.0`)
+- [`puppetlabs-mysql`][puppetlabs-mysql] (`10.10.0`)
+- [`puppetlabs-tagmail`][puppetlabs-tagmail] (`3.5.1`)
+
+  [puppetlabs-apt]: https://github.com/puppetlabs/puppetlabs-apt
+  [puppetlabs-apache]: https://github.com/puppetlabs/puppetlabs-apache
+  [puppetlabs-mysql]: http://github.com/puppetlabs/puppetlabs-mysql
+  [puppetlabs-tagmail]: https://github.com/puppetlabs/puppetlabs-tagmail/
+
```
Comment (sanfrancrisko) [inline _posts/2021-02-19-status-update.md:52]:
```suggestion

## Customer Defects Addressed

[MODULES-10899][MODULES-10899]: Thanks to [cdamage][cdamage] for liaising with the original reporter of this issue and getting a PR with a fix ready so promptly. 
We'd also like to thank [beatchristen][beatchristen] for additional contributions for this PR to allow PHP 8 packages to be installed on Debian 10 when using the experimental repositories.
Finally, thanks to [gguillotte][gguillotte] for reviewing and helping to get it over the line.
```

Links:
```md
[MODULES-10899]: https://tickets.puppetlabs.com/browse/MODULES-10899
```

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]
Also not sure why the links have been dropped out of the script? Were they pulled out for any particular reason?

File: _posts/2021-02-19-status-update.md (hunk 2 above)
Comment (carabasdaniel) [reply _posts/2021-02-19-status-update.md:58]: Added details as part of the Community Contributions

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]
I'm still not seeing the links to the PRs in here. E.g. `puppetlabs-apache-pr-2120`

**Comment by DavidS** [comment]
- [ ] IAC-1414 ?
- [ ] tomcat install issues?
- [ ] there were a few helm/kube tix resolved this week, was that just cleanup of old tix or can we mention the work done?

**Comment by sanfrancrisko** [comment]
> * [ ]  [IAC-1414](https://tickets.puppetlabs.com/browse/IAC-1414) ?
> * [ ]  tomcat install issues?
> * [ ]  there were a few helm/kube tix resolved this week, was that just cleanup of old tix or can we mention the work done?

@DavidS IAC-1414 was included in last week's blog.

**Comment by carabasdaniel** [comment]
> * [x]  [IAC-1414](https://tickets.puppetlabs.com/browse/IAC-1414) ?
> * [ ]  tomcat install issues?
> * [ ]  there were a few helm/kube tix resolved this week, was that just cleanup of old tix or can we mention the work done?

For tomcat it was a maintenance commit to fix acceptance tests and for helm it was a bugfix that updates the default stable repository but has not been released yet.

**Review by adrianiurca — APPROVED** [review]

**Pull request merged**